keychain/src/methods/register.ts

```diff
@@ -43,7 +43,14 @@ const register =
     controller.store();
     Storage.set(selectors["0.0.3"].active(), address);
 
-    return { address, deviceKey };
+    return { address, deviceKey, keypair };
   };
 
-export default register;
+const saveDeploy = (address: string, hash: string) => {
```

Comment:
we can add this to the controller interface here:

https://github.com/cartridge-gg/cartridge/blob/main/controller/src/types.ts#L38